### PR TITLE
[DATASTD-1441] ProviderCategory descriptors should be aligned with Ed-Fi standard practice and defined

### PR DESCRIPTION
**Pull request by ggaronzik**

### CODE REVIEW

File: Descriptors/ProviderCategoryDescriptor.xml [hunk 1]
```diff
@@ -1,45 +1,129 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <InterchangeDescriptors xmlns="http://ed-fi.org/3.3.0-b" xmlns:ann="http://ed-fi.org/annotation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://ed-fi.org/3.3.0-b ../Schemas/Bulk/Interchange-Descriptors.xsd">
+   <ProviderCategoryDescriptor>
```
Comment (blmeyers) [inline Descriptors/ProviderCategoryDescriptor.xml:3]:
```suggestion
	<ProviderCategoryDescriptor>
```

**Review by blmeyers — CHANGES_REQUESTED** [review]
Value changes look fine, but formatting seems to have gotten off.

File: Descriptors/ProviderCategoryDescriptor.xml (hunk 1 above)
Comment (ggaronzik) [reply Descriptors/ProviderCategoryDescriptor.xml]: formatting fixed!

**Review by blmeyers — APPROVED** [review]

**Review by ejansson — CHANGES_REQUESTED** [review]
Other should be added as a new category and Not applicable marked as deprecated.

File: Descriptors/ProviderCategoryDescriptor.xml [hunk 2]
```diff
@@ -1,45 +1,129 @@
...
+		<Namespace>uri://ed-fi.org/ProviderCategoryDescriptor</Namespace>
+	</ProviderCategoryDescriptor>
+	<ProviderCategoryDescriptor>
+		<CodeValue>On-premise child care</CodeValue>
+		<ShortDescription>On-premise child care</ShortDescription>
+		<Description>On-premise child care</Description>
+		<Namespace>uri://ed-fi.org/ProviderCategoryDescriptor</Namespace>
+	</ProviderCategoryDescriptor>
+	<ProviderCategoryDescriptor>
+		<CodeValue>Other</CodeValue>
```
Comment (ggaronzik) [inline Descriptors/ProviderCategoryDescriptor.xml:52]: @ejansson here is where Other is added as a new ProviderCategory.

File: Descriptors/ProviderCategoryDescriptor.xml [hunk 3]
```diff
@@ -1,45 +1,129 @@
...
+		<ShortDescription>DEPRECATED: Licensed Large Family Child Care</ShortDescription>
+		<Description>DEPRECATED: Licensed Large Family Child Care</Description>
 		<Namespace>uri://ed-fi.org/ProviderCategoryDescriptor</Namespace>
 	</ProviderCategoryDescriptor>
 	<ProviderCategoryDescriptor>
 		<CodeValue>Not Applicable</CodeValue>
-		<ShortDescription>Not Applicable</ShortDescription>
-		<Description>Not Applicable</Description>
+		<ShortDescription>DEPRECATED: Not Applicable</ShortDescription>
+		<Description>DEPRECATED: Not Applicable</Description>
```
Comment (ggaronzik) [inline Descriptors/ProviderCategoryDescriptor.xml:126]: @ejansson here is where Not Applicable is deprecated.

**Comment by ggaronzik** [comment]
> Other should be added as a new category and Not applicable marked as deprecated.

@ejansson I believe these are covered. I tagged you in a comment for each. Let me know if there are other changes needed.

**Review by ejansson — APPROVED** [review]

**Pull request merged**